torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -739,22 +744,13 @@ void InplaceConverter::replaceAttrWithInplaceOps(
     TORCH_INTERNAL_ASSERT(
         n->kind() == prim::GetAttr || n->kind() == prim::SetAttr);
     if (n->kind() == prim::SetAttr) {
-      // Convert SetAttr to inplace op.
-      // Directly convert to index_put_ instead of copy_, since we know expand
-      // is not required for value.
+      // Convert SetAttr to inplace op aten::set_.
```

Comment:
Is there something similar for getAttr?

Reply:
For GetAttr it is only the value itself, after we have flattened the module structure, and have access to the value. Can be viewed as equivalent to Identity, but optimized away.